tests/acceptance/helpers/filesFoldersHelpers.js

```diff
@@ -0,0 +1,50 @@
+const { client } = require('nightwatch-api')
+const { propfind } = require('./webdavHelper')
+const convert = require('xml-js')
+const { relativeTo, normalize, join } = require('./path')
+
+exports.getAllFilesFolders = function () {
+  const user = client.globals.currentUser
```

Comment:
I think it would be better to have the username here as a parameter to the function, so we can use it also with an other user than the currently logged in UI user

Reply:
fixed